scripts/remove-tagged-compat-features.ts

```diff
@@ -0,0 +1,118 @@
+import { setLogger } from "compute-baseline";
+import { Compat } from "compute-baseline/browser-compat-data";
```

Comment:
Actually this seems to be used in line 40

```suggestion
import { Compat, Feature } from "compute-baseline/browser-compat-data";
```